pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,35 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        """Test that the errors=ignore argument silences the exception that is
+        thrown by default for data that cannot be converted to the specified
+        type with astype.
+
+        Test for GH-25905
+        """
+        data_df = pd.DataFrame([{'a': '1',
```

Comment:
w.r.t to formatting if you break the line before the opening dict here should be able to make this a lot more conscise (essentially one dict per line)

Reply:
This breaks the style of the rest of the code in the file. Would you still like this change?

Reply:
Yes

Reply:
Formatted in f2dd79bf97a0c640a77339dfc0a25406b5a9581a